FrameworkSystem/test/testLogging/tests/Test_DisplayOptions.py

```diff
@@ -249,7 +249,7 @@ def test_07subLogShowHeadersChange(self):
     self.assertEqual(message, "message \nmessage \n")
     self.assertEqual(self.buffer.getvalue(), "message \n")
 
-  def test_07subLogShowHeadersChangeSetSubLogger(self):
+  def test_08subLogShowHeadersChangeSetSubLogger(self):
```

Comment:
I have done somewhat "similar" fixes in https://github.com/DIRACGrid/DIRAC/pull/3387/files#diff-57a8d766a2b523fe8ab265a391860f76 but it looks to me what you have done here is more correct. In order to avoid conflicts, I will remove the changes I have done in my PR.